sqlglot/dialects/prql.py

```diff
@@ -104,8 +104,18 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_ordered(
+            self, parse_method: t.Optional[t.Callable] = None
+        ) -> t.Optional[exp.Ordered]:
+            asc = self._match(TokenType.PLUS)
+            desc = self._match(TokenType.DASH) or (asc and False)
+            term = super()._parse_ordered(parse_method)
+            if term and desc:
+                return term.desc()
```

Comment:
This will create an `Ordered` under an `Ordered`, look into the definition of `desc`: https://github.com/tobymao/sqlglot/blob/b523bb630b28866ebab581d43e99f0b2b821ec12/sqlglot/expressions.py#L908-L909

You need to do `term.set("desc", True)` instead if `term and desc` here.